### PR TITLE
DR-2996 UI updates minor fixups

### PR DESCRIPTION
**Pull request by nmalfroy**
Ran:
`npm upgrade` + `npm update` to pull in the latest minor updates to our libraries

A couple of drive-by changes (can be viewed commit-by-commit for more details) were needed to make the tests pass

I also updated the dependabot config to ignore all major version of mui and added the cypress base library to the ignore list

I updated the readme to hopefully make it a little easier to run e2e tests

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-  open-pull-requests-limit: 10
-  ignore:
-  # newer versions of prettier seem to format the code in a way that is incompatible with eslint (DR-2953)
-  - dependency-name: "prettier"
-  # newer versions of material ui (MUI) cause our styling to not respect theming or straight up break the UI (DR-2952)
-  - dependency-name: "mui"
-  # new versions prevents unit tests from running (DR-2954)
-  - dependency-name: "cypress/react"
-
+  - package-ecosystem: npm
```
Comment (okotsopoulos) [inline .github/dependabot.yml]: Is there any reason this got reformatted? Would be great to keep existing formatting if that's okay, to preserve the git blame.

**Review by snf2ye — APPROVED** [review]

**Review by okotsopoulos — APPROVED** [review]
I count 16 dependency updates (fewer than I'd expect, honestly)… and 3k line changes.  Whew!  Thanks so much for working on this!

**Comment by nmalfroy** [comment]
Haha, easiest 3K line change ever :)

File: .github/dependabot.yml (hunk 1 above)
Comment (nmalfroy) [reply .github/dependabot.yml]: Oh yep!  Sorry,  don't know why it decided to refactor everything.  Lemme see if I can remove the whitespace changes

**Pull request merged**